docs/docs-beta/docs/guides/data-modeling/data-assets.md

```diff
@@ -2,3 +2,111 @@
 title: "Create data assets"
 sidebar_position: 10
 ---
+
+Data assets are at the core of Dagster. By modelling your data as assets you unlock: declarative automation, lineage, and a searchable data catalog. This guide will help you understand how to model your data as assets.
+
+There are three ways of create data assets:
+
+* Using the `@asset` decorator to declare a single asset.
+* Using the `@multi_asset` decorator to output multiple assets from a single operation.
+* Using the `@graph_asset` decorator to output a single asset from multiple operations without making each operation itself an asset.
+
+This guide walks through all three methods.
+
+***
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+* A basic understanding of Dagster concepts such as assets and resources
+* Dagster installed and a working project setup
+
+</details>
+
+***
+
+## Define a single data asset using the `@asset` decorator
+
+The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.
```

Comment:
I don't think we want to default to this approach which invokes secret IO Managers. 

```suggestion
The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.

This should also be a code snippet file so it can be tested/formatted. 

```python
from dagster import dg

@dg.asset
def my_data_asset(context: AssetExecutionContext):
    context.log.info("Hello from my asset")
```

Reply:
By "this approach" you mean "@dg.asset" ? What should the default be?

Reply:
ah I mean the `return a simple dictionary` for an asset.

Instead we default to not passing values through assets until you have learned about IO managers.